File: app/src/main/java/com/example/xeltim/biologyjourney/Content.java
package com.example.xeltim.biologyjourney;
/**
 * Created by CiulacuCodrut on 11/26/2017.
 */

public class Content {

    private String title;
    private String[] url;

    public Content(String[] url, String title){
        this.url = url;
        this.title = title;
    }

    public String[] getURL(){ return url; }
    public String getTitle(){ return title; }

}

File: app/src/main/java/com/example/xeltim/biologyjourney/Content.kt
package com.example.xeltim.biologyjourney

class Content(var URL: Array<String>, var title: String?){

    private var mTitle : String? = null
    get() = mTitle

    private var mURL : Array<String>? = null
    get() = mURL

    init{
        this.mTitle = title
        this.mURL = URL
    }



}